lib/app.js

```diff
@@ -61,6 +62,13 @@ app.post('/cloud/:datasetId', function(req, res) {
 var excludedPaths = ['/authpolicy', '/file'];
 app.use(require('fh-wfm-user/lib/middleware/validateSession')(mediator, mbaasApi,excludedPaths));
 
+
+//Initialising the fh-wfm-sync data sets for each module
+wfmSync.init(mediator, mbaasApi, 'workorders', config.get("syncOptions"));
```

Comment:
Can we use this as array in the config so users would be able to init different datasets and then we can document this to make this setup easier. If we move here from app perspective to framework perspective and then describe how to add new dataset to the game.